src/app/profile/spaces/spaces.component.html

```diff
@@ -24,7 +24,7 @@ <h2>{{context?.user?.attributes?.fullName}}</h2>
           <div class="form-group">
             <div class="col-sm-10">
               <div class="search-pf-input-group">
-                <input id="spacesSearch" type="search" class="form-control" placeholder="Search spaces..." #search (keyup)="searchSpaces(search.value)">
+                <input id="spacesSearch" type="search" class="form-control" placeholder="Search spaces..." #search (keyup)="searchSpaces(search.value)" (keydown.enter)="$event.preventDefault()">
```

Comment:
Instead of preventing the enter key, perhaps we should prevent the form submit? Or, remove the unused form tag all together.

Option 1 (prevent the form submit):
`<form class="form-horizontal" onsubmit="return false;">`

Option 2 (change the form to a div):
`<div class="form-horizontal">`

Reply:
Makes sense. I've gone for #2 in the revised commit.